src/standalone/Bucket/Bucket.jl

```diff
@@ -553,6 +553,31 @@ function ClimaLand.get_drivers(model::BucketModel)
     return (model.atmos, model.radiation)
 end
 
+"""
+    ClimaLand.surface_air_density(
+                    atmos::CoupledAtmosphere,
+                    model::AbstractModel,
+                    Y,
+                    p,
+                    _...,
+                )
+Returns the air density at the surface in the case of a coupled simulation.
+
+This requires the field `ρ_sfc` to be present in the cache `p` under the name
+of the model.
+"""
+function surface_air_density(
+    atmos::CoupledAtmosphere,
+    model::BucketModel,
+    Y,
+    p,
+    _...,
+)
+    model_name = ClimaLand.name(model)
+    model_cache = getproperty(p, model_name)
+    return model_cache.ρ_sfc
```

Comment:
We can simplify this since it is not bucket specific - `return p.bucket.rho_sfc`?